2024-02-02 17:31:47,453:DEBUG:certbot._internal.main:certbot version: 2.8.0
2024-02-02 17:31:47,454:DEBUG:certbot._internal.main:Location of certbot entry point: /usr/local/bin/certbot
2024-02-02 17:31:47,454:DEBUG:certbot._internal.main:Arguments: ['--webroot', '--webroot-path=/usr/share/nginx/html/letsencrypt', '--email', 'desarrollo.jadalsoftware@gmail.com', '--agree-tos', '--no-eff-email', '-d', 'migracion.jadal.pe']
2024-02-02 17:31:47,454:DEBUG:certbot._internal.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#manual,PluginEntryPoint#null,PluginEntryPoint#standalone,PluginEntryPoint#webroot)
2024-02-02 17:31:47,475:DEBUG:certbot._internal.log:Root logging level set at 30
2024-02-02 17:31:47,476:DEBUG:certbot._internal.plugins.selection:Requested authenticator webroot and installer None
2024-02-02 17:31:47,476:DEBUG:certbot._internal.plugins.selection:Single candidate plugin: * webroot
Description: Saves the necessary validation files to a .well-known/acme-challenge/ directory within the nominated webroot path. A seperate HTTP server must be running and serving files from the webroot path. HTTP challenge only (wildcards not supported).
Interfaces: Authenticator, Plugin
Entry point: EntryPoint(name='webroot', value='certbot._internal.plugins.webroot:Authenticator', group='certbot.plugins')
Initialized: <certbot._internal.plugins.webroot.Authenticator object at 0x7f04c8ad4a30>
Prep: True
2024-02-02 17:31:47,476:DEBUG:certbot._internal.plugins.selection:Selected authenticator <certbot._internal.plugins.webroot.Authenticator object at 0x7f04c8ad4a30> and installer None
2024-02-02 17:31:47,477:INFO:certbot._internal.plugins.selection:Plugins selected: Authenticator webroot, Installer None
2024-02-02 17:31:47,543:DEBUG:certbot._internal.main:Picked account: <Account(RegistrationResource(body=Registration(key=None, contact=(), agreement=None, status=None, terms_of_service_agreed=None, only_return_existing=None, external_account_binding=None), uri='https://acme-v02.api.letsencrypt.org/acme/acct/1550272667', new_authzr_uri=None, terms_of_service=None), 9b507c65556e201cf96f44fb1f74216d, Meta(creation_dt=datetime.datetime(2024, 2, 2, 17, 5, 57, tzinfo=<UTC>), creation_host='ee38a32c6a85', register_to_eff=None))>
2024-02-02 17:31:47,544:DEBUG:acme.client:Sending GET request to https://acme-v02.api.letsencrypt.org/directory.
2024-02-02 17:31:47,546:DEBUG:urllib3.connectionpool:Starting new HTTPS connection (1): acme-v02.api.letsencrypt.org:443
2024-02-02 17:31:47,712:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "GET /directory HTTP/1.1" 200 752
2024-02-02 17:31:47,713:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 17:31:47 GMT
Content-Type: application/json
Content-Length: 752
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "keyChange": "https://acme-v02.api.letsencrypt.org/acme/key-change",
  "meta": {
    "caaIdentities": [
      "letsencrypt.org"
    ],
    "termsOfService": "https://letsencrypt.org/documents/LE-SA-v1.3-September-21-2022.pdf",
    "website": "https://letsencrypt.org"
  },
  "newAccount": "https://acme-v02.api.letsencrypt.org/acme/new-acct",
  "newNonce": "https://acme-v02.api.letsencrypt.org/acme/new-nonce",
  "newOrder": "https://acme-v02.api.letsencrypt.org/acme/new-order",
  "renewalInfo": "https://acme-v02.api.letsencrypt.org/draft-ietf-acme-ari-02/renewalInfo/",
  "revokeCert": "https://acme-v02.api.letsencrypt.org/acme/revoke-cert",
  "x8QHI-TJjcI": "https://community.letsencrypt.org/t/adding-random-entries-to-the-directory/33417"
}
2024-02-02 17:31:47,713:DEBUG:certbot._internal.display.obj:Notifying user: Requesting a certificate for migracion.jadal.pe
2024-02-02 17:31:47,727:DEBUG:acme.client:Requesting fresh nonce
2024-02-02 17:31:47,727:DEBUG:acme.client:Sending HEAD request to https://acme-v02.api.letsencrypt.org/acme/new-nonce.
2024-02-02 17:31:47,786:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "HEAD /acme/new-nonce HTTP/1.1" 200 0
2024-02-02 17:31:47,786:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 17:31:47 GMT
Connection: keep-alive
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 5IXlosaqckRMTszVqarAgA1vpyoUPmPHQupchcj1lPbQ_OKjUDI
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800


2024-02-02 17:31:47,786:DEBUG:acme.client:Storing nonce: 5IXlosaqckRMTszVqarAgA1vpyoUPmPHQupchcj1lPbQ_OKjUDI
2024-02-02 17:31:47,787:DEBUG:acme.client:JWS payload:
b'{\n  "identifiers": [\n    {\n      "type": "dns",\n      "value": "migracion.jadal.pe"\n    }\n  ]\n}'
2024-02-02 17:31:47,790:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/new-order:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICI1SVhsb3NhcWNrUk1Uc3pWcWFyQWdBMXZweW9VUG1QSFF1cGNoY2oxbFBiUV9PS2pVREkiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL25ldy1vcmRlciJ9",
  "signature": "bRdy6OJJQDO0MB6HFVq50Z4on3eBNLVAjrpy38Fq5Aa9MPmy5feukW5yVvxcPM9H0jA9Bpf4Y3PtB01cqrTleHeLT8tcq1ljwv1vcpfHi2IxC13AOnxvgXn1Gwc2nWcBT5oB3ZfvnnWm-HNxZEetyMzfx2ortLpqk3AO3d9hJ9VGem1k3q4FnUDB0CJIXVfy0BpUHZX4qvqKqFzWsW5_hR8Ey_df4FIKZf0NeFXuyHjCIT2d18J1Y35fs2spImiyEdUdx9xsCRFmB9pGc1mdu0IZoMvJ-RbZz5QWSR5Qt7osPv6kHEnzmzCwYipdV63pOU8EgBec32GjM2QdutlUPw",
  "payload": "ewogICJpZGVudGlmaWVycyI6IFsKICAgIHsKICAgICAgInR5cGUiOiAiZG5zIiwKICAgICAgInZhbHVlIjogIm1pZ3JhY2lvbi5qYWRhbC5wZSIKICAgIH0KICBdCn0"
}
2024-02-02 17:31:48,027:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/new-order HTTP/1.1" 201 344
2024-02-02 17:31:48,028:DEBUG:acme.client:Received response:
HTTP 201
Server: nginx
Date: Fri, 02 Feb 2024 17:31:47 GMT
Content-Type: application/json
Content-Length: 344
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Location: https://acme-v02.api.letsencrypt.org/acme/order/1550272667/241333224337
Replay-Nonce: 3l89SrA7sugpR13DU65qPLLqY-gdti-5ma3GtgkVRPlIAgWMH5s
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "status": "pending",
  "expires": "2024-02-09T17:31:47Z",
  "identifiers": [
    {
      "type": "dns",
      "value": "migracion.jadal.pe"
    }
  ],
  "authorizations": [
    "https://acme-v02.api.letsencrypt.org/acme/authz-v3/310854801507"
  ],
  "finalize": "https://acme-v02.api.letsencrypt.org/acme/finalize/1550272667/241333224337"
}
2024-02-02 17:31:48,028:DEBUG:acme.client:Storing nonce: 3l89SrA7sugpR13DU65qPLLqY-gdti-5ma3GtgkVRPlIAgWMH5s
2024-02-02 17:31:48,029:DEBUG:acme.client:JWS payload:
b''
2024-02-02 17:31:48,030:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/310854801507:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICIzbDg5U3JBN3N1Z3BSMTNEVTY1cVBMTHFZLWdkdGktNW1hM0d0Z2tWUlBsSUFnV01INXMiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMxMDg1NDgwMTUwNyJ9",
  "signature": "UCznr86A28T-jPTs0pE7W2J-MiuZUs1dZuP3vd9evGMR54RsvZLD43KV25L4WzhFTIa0RlkDtWzSyiIolddx-_pvMYr4y_t1cZzdpbrneZeueuoPEXOCdtsA60gAw8VVsR_Hn2u6s8OtF4uSJxHkfGmdBZb_L3u4_I795Ep5vO7p9V8p72N9guM_cycWtAWX_kHTIEmxW-XApndgRgNEdCz3QmeQ2En9nLtwQeyaXXwUnlXwYwiiyTj5iYRogqblhmlg3v8ieSD3Jbi5l1Mi7H4hp7LLmUx9kbQxlhvyJszKpXtlFuCRvbvbgn89JPopftclnvr1xrRGmcYyMXpzAQ",
  "payload": ""
}
2024-02-02 17:31:48,136:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/310854801507 HTTP/1.1" 200 802
2024-02-02 17:31:48,136:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 17:31:48 GMT
Content-Type: application/json
Content-Length: 802
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 5IXlosaqPprY7mTDoeiNPcX4z9iefvH5-QNRjXWfog2Ylo8hktg
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "migracion.jadal.pe"
  },
  "status": "pending",
  "expires": "2024-02-09T17:31:47Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310854801507/y9-YXg",
      "token": "zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310854801507/4hE3Zw",
      "token": "zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310854801507/7coSfA",
      "token": "zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU"
    }
  ]
}
2024-02-02 17:31:48,136:DEBUG:acme.client:Storing nonce: 5IXlosaqPprY7mTDoeiNPcX4z9iefvH5-QNRjXWfog2Ylo8hktg
2024-02-02 17:31:48,137:INFO:certbot._internal.auth_handler:Performing the following challenges:
2024-02-02 17:31:48,137:INFO:certbot._internal.auth_handler:http-01 challenge for migracion.jadal.pe
2024-02-02 17:31:48,137:INFO:certbot._internal.plugins.webroot:Using the webroot path /usr/share/nginx/html/letsencrypt for all unmatched domains.
2024-02-02 17:31:48,137:DEBUG:certbot._internal.plugins.webroot:Creating root challenges validation dir at /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge
2024-02-02 17:31:48,138:DEBUG:certbot._internal.plugins.webroot:Attempting to save validation to /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU
2024-02-02 17:31:48,139:DEBUG:acme.client:JWS payload:
b'{}'
2024-02-02 17:31:48,140:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/chall-v3/310854801507/y9-YXg:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICI1SVhsb3NhcVBwclk3bVREb2VpTlBjWDR6OWllZnZINS1RTlJqWFdmb2cyWWxvOGhrdGciLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2NoYWxsLXYzLzMxMDg1NDgwMTUwNy95OS1ZWGcifQ",
  "signature": "bCpXeeJeaOq5m-a4tJ1I2KBUCC_AM2dE4nldD9B7qtcUKviEtSXI9paVTvA_2FOUdnbHkU58FQHdDvnVLkSQskLrmJVeZEvuypsDyLo_5o0k28YSHP14w78Dpf2w-iBiSYDPufqkjUGGRQfJP17g1dxl98pphepyFAE9OBoQ7pROThG-wCIPKoSOIT-nh-9Zhf5_3AtslID_yDLepn7fkFvAlugizOKAlOHDwYGOqnW19g3cQ0ZiS8IB0fCUrTzyK_Slp3J3kTsuQ8M0epI69YtSvHdxx_dp7ssjhfJOzY_eGDVqJy4Xh9G9Vb5a1nRq40apT0_g2MVjptmf1LrpEQ",
  "payload": "e30"
}
2024-02-02 17:31:48,202:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/chall-v3/310854801507/y9-YXg HTTP/1.1" 200 187
2024-02-02 17:31:48,202:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 17:31:48 GMT
Content-Type: application/json
Content-Length: 187
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index", <https://acme-v02.api.letsencrypt.org/acme/authz-v3/310854801507>;rel="up"
Location: https://acme-v02.api.letsencrypt.org/acme/chall-v3/310854801507/y9-YXg
Replay-Nonce: 3l89SrA7sQfTNjWtRIk9mHP7GJucx72Nesl-CAegNVWKmus1OK4
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "type": "http-01",
  "status": "pending",
  "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310854801507/y9-YXg",
  "token": "zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU"
}
2024-02-02 17:31:48,202:DEBUG:acme.client:Storing nonce: 3l89SrA7sQfTNjWtRIk9mHP7GJucx72Nesl-CAegNVWKmus1OK4
2024-02-02 17:31:48,203:INFO:certbot._internal.auth_handler:Waiting for verification...
2024-02-02 17:31:49,204:DEBUG:acme.client:JWS payload:
b''
2024-02-02 17:31:49,206:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/310854801507:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICIzbDg5U3JBN3NRZlROald0UklrOW1IUDdHSnVjeDcyTmVzbC1DQWVnTlZXS211czFPSzQiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMxMDg1NDgwMTUwNyJ9",
  "signature": "tCq3SUDvOxyj2KBIStD0BhSrz1hTbM6EeSV5YAM26YGWAge_wvhtTtFvtoy64E_F3B4b6CBqSYUbf6Hrl6Peh4Nfbalfqe93lb3BWbLf6ekXYHOYPvdU97R88hVyl8vZ7WND4xIDooeEIAZ5sSrRd1gUE4mPTq-fA3De5CIHaU11fk--RIDoMaaDVvG4dPoWCqA6QxfL_XtCJWc23PWh2hgbqib7Io-Ilj9GIRP9D-aA4RPW1BfOK3_sVLWxjpoz3Lf7UTVvhrNLeWNuSsCBNUrN8D7M4mCZv0u4fFbXR0Ia1YWs8G3T4NuPaZ7OQsIR6ongL626hPR1QNORdptGjw",
  "payload": ""
}
2024-02-02 17:31:49,277:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/310854801507 HTTP/1.1" 200 802
2024-02-02 17:31:49,277:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 17:31:49 GMT
Content-Type: application/json
Content-Length: 802
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 3l89SrA7UICj0oi9tntpKrmXPwrCm68I9IFZXlcOvcBM9cL1XiY
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "migracion.jadal.pe"
  },
  "status": "pending",
  "expires": "2024-02-09T17:31:47Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310854801507/y9-YXg",
      "token": "zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU"
    },
    {
      "type": "dns-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310854801507/4hE3Zw",
      "token": "zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU"
    },
    {
      "type": "tls-alpn-01",
      "status": "pending",
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310854801507/7coSfA",
      "token": "zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU"
    }
  ]
}
2024-02-02 17:31:49,277:DEBUG:acme.client:Storing nonce: 3l89SrA7UICj0oi9tntpKrmXPwrCm68I9IFZXlcOvcBM9cL1XiY
2024-02-02 17:31:52,281:DEBUG:acme.client:JWS payload:
b''
2024-02-02 17:31:52,282:DEBUG:acme.client:Sending POST request to https://acme-v02.api.letsencrypt.org/acme/authz-v3/310854801507:
{
  "protected": "eyJhbGciOiAiUlMyNTYiLCAia2lkIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2FjY3QvMTU1MDI3MjY2NyIsICJub25jZSI6ICIzbDg5U3JBN1VJQ2owb2k5dG50cEtybVhQd3JDbTY4STlJRlpYbGNPdmNCTTljTDFYaVkiLCAidXJsIjogImh0dHBzOi8vYWNtZS12MDIuYXBpLmxldHNlbmNyeXB0Lm9yZy9hY21lL2F1dGh6LXYzLzMxMDg1NDgwMTUwNyJ9",
  "signature": "T3C9758YtAZOMlV0Gy9jDkBXqRH7bdTLK54rRdBJFJfPWjAAhVkTuc284jy6q_iS6Bf2wnKmdUIekrwu8UgZVvBD5QEYCZRzQLzsMthEylpZSsPmBDBnAxHm_g7QxQmkobO7NEnQaucdrZYpnpMkd4BHr3GeR-dDVRMHeoP4jDWLYanS9bEhAZBd45dz0jmz-XhVfn1IvaJ-fuevE_Sf8hEYoXCnyErKV715ysN-33ugcmH5pxTt7TpH9UaEFTvc_pz-zy3-A4kGQ-yuEQjzOXUYJkzUrjfdI-2nMJmN5DxBhPU-zi4snttVZkO5dppHXIFOlgrMsfRIF3oCQvj_lQ",
  "payload": ""
}
2024-02-02 17:31:52,340:DEBUG:urllib3.connectionpool:https://acme-v02.api.letsencrypt.org:443 "POST /acme/authz-v3/310854801507 HTTP/1.1" 200 1354
2024-02-02 17:31:52,340:DEBUG:acme.client:Received response:
HTTP 200
Server: nginx
Date: Fri, 02 Feb 2024 17:31:52 GMT
Content-Type: application/json
Content-Length: 1354
Connection: keep-alive
Boulder-Requester: 1550272667
Cache-Control: public, max-age=0, no-cache
Link: <https://acme-v02.api.letsencrypt.org/directory>;rel="index"
Replay-Nonce: 3l89SrA7TQci_UYxa3YBQoI1uBmh_VVVSkv_W1mxwqNbYgeM4CQ
X-Frame-Options: DENY
Strict-Transport-Security: max-age=604800

{
  "identifier": {
    "type": "dns",
    "value": "migracion.jadal.pe"
  },
  "status": "invalid",
  "expires": "2024-02-09T17:31:47Z",
  "challenges": [
    {
      "type": "http-01",
      "status": "invalid",
      "error": {
        "type": "urn:ietf:params:acme:error:unauthorized",
        "detail": "3.85.216.85: Invalid response from https://migracion.jadal.pe/.well-known/acme-challenge/zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU: 404",
        "status": 403
      },
      "url": "https://acme-v02.api.letsencrypt.org/acme/chall-v3/310854801507/y9-YXg",
      "token": "zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU",
      "validationRecord": [
        {
          "url": "http://migracion.jadal.pe/.well-known/acme-challenge/zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU",
          "hostname": "migracion.jadal.pe",
          "port": "80",
          "addressesResolved": [
            "3.85.216.85"
          ],
          "addressUsed": "3.85.216.85"
        },
        {
          "url": "https://migracion.jadal.pe/.well-known/acme-challenge/zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU",
          "hostname": "migracion.jadal.pe",
          "port": "443",
          "addressesResolved": [
            "3.85.216.85"
          ],
          "addressUsed": "3.85.216.85"
        }
      ],
      "validated": "2024-02-02T17:31:48Z"
    }
  ]
}
2024-02-02 17:31:52,340:DEBUG:acme.client:Storing nonce: 3l89SrA7TQci_UYxa3YBQoI1uBmh_VVVSkv_W1mxwqNbYgeM4CQ
2024-02-02 17:31:52,341:INFO:certbot._internal.auth_handler:Challenge failed for domain migracion.jadal.pe
2024-02-02 17:31:52,341:INFO:certbot._internal.auth_handler:http-01 challenge for migracion.jadal.pe
2024-02-02 17:31:52,341:DEBUG:certbot._internal.display.obj:Notifying user: 
Certbot failed to authenticate some domains (authenticator: webroot). The Certificate Authority reported these problems:
  Domain: migracion.jadal.pe
  Type:   unauthorized
  Detail: 3.85.216.85: Invalid response from https://migracion.jadal.pe/.well-known/acme-challenge/zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU: 404

Hint: The Certificate Authority failed to download the temporary challenge files created by Certbot. Ensure that the listed domains serve their content from the provided --webroot-path/-w and that files created there can be downloaded from the internet.

2024-02-02 17:31:52,343:DEBUG:certbot._internal.error_handler:Encountered exception:
Traceback (most recent call last):
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.

2024-02-02 17:31:52,343:DEBUG:certbot._internal.error_handler:Calling registered functions
2024-02-02 17:31:52,343:INFO:certbot._internal.auth_handler:Cleaning up challenges
2024-02-02 17:31:52,343:DEBUG:certbot._internal.plugins.webroot:Removing /usr/share/nginx/html/letsencrypt/.well-known/acme-challenge/zVh9fWnZFHIX4kdR__LIrv6TQuwTRugKSnbBtYeVfEU
2024-02-02 17:31:52,344:DEBUG:certbot._internal.plugins.webroot:All challenges cleaned up
2024-02-02 17:31:52,344:DEBUG:certbot._internal.log:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/local/bin/certbot", line 33, in <module>
    sys.exit(load_entry_point('certbot', 'console_scripts', 'certbot')())
  File "/opt/certbot/src/certbot/certbot/main.py", line 19, in main
    return internal_main.main(cli_args)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1869, in main
    return config.func(config, plugins)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 1600, in certonly
    lineage = _get_and_save_cert(le_client, config, domains, certname, lineage)
  File "/opt/certbot/src/certbot/certbot/_internal/main.py", line 143, in _get_and_save_cert
    lineage = le_client.obtain_and_enroll_certificate(domains, certname)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 517, in obtain_and_enroll_certificate
    cert, chain, key, _ = self.obtain_certificate(domains)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 428, in obtain_certificate
    orderr = self._get_order_and_authorizations(csr.data, self.config.allow_subset_of_names)
  File "/opt/certbot/src/certbot/certbot/_internal/client.py", line 496, in _get_order_and_authorizations
    authzr = self.auth_handler.handle_authorizations(orderr, self.config, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 108, in handle_authorizations
    self._poll_authorizations(authzrs, max_retries, max_time_mins, best_effort)
  File "/opt/certbot/src/certbot/certbot/_internal/auth_handler.py", line 212, in _poll_authorizations
    raise errors.AuthorizationError('Some challenges have failed.')
certbot.errors.AuthorizationError: Some challenges have failed.
2024-02-02 17:31:52,353:ERROR:certbot._internal.log:Some challenges have failed.
